packages/flutter_tools/templates/app/android.tmpl/.gitignore

```diff
@@ -5,3 +5,4 @@ gradle-wrapper.jar
 /gradlew.bat
 /local.properties
 GeneratedPluginRegistrant.java
+key.properties
```

Comment:
I'd maybe add a line of comment here to just say that this was added as a "reminder". You can obviously check your key into your (ideally private) repo if you want. It's not "unclean" unlike say GeneratedPluginRegistrant.java.

Reply:
Good idea.  Is that comment okay?

Reply:
LG